### PR TITLE
Fix env var name for runner creation

### PR DESCRIPTION
**Pull request by sidpremkumar**
## Description of the change

This ensures we use our `IN_PROD` flag (so we can use it in dev + prod) in order to create all the targets across all operating systems

## Testing

Describe how to test this PR....

**backend branch:** 
**bzero branch:** 

#### Ready to run system tests?

- [ ] Yes

## Relevant release note information

Release Notes: Fixup System test runner creation logic

## Related JIRA tickets

Relates to JIRA: CWC-1603

## Relevant HotFix if applicable

Relevant HotFix PR Number:

## Have you considered the security impacts?

Does this PR have any security impact?

- [ ] Yes
- [ ] No

If yes, please explain:

### CODE REVIEW

**Review by ymarcus93 — CHANGES_REQUESTED** [review]
Isn't the python wrapper adding ap-northeast-1 to EXTRA_REGIONS only in prod? I'm not exactly sure what change we're making here in zli project.

**Comment by sidpremkumar** [comment]
> Isn't the python wrapper adding ap-northeast-1 to EXTRA_REGIONS only in prod? I'm not exactly sure what change we're making here in zli project.

yeah I made a mistake, it just fixes the issue where we are not building all targets in the pipeline

**Comment by ymarcus93** [comment]
Oh sorry. I just saw the PR title changed. My bad

File: src/system-tests/tests/system-test.ts [hunk 1]
```diff
@@ -65,6 +65,7 @@ export const KUBE_ENABLED = process.env.KUBE_ENABLED ? (process.env.KUBE_ENABLED
 const VT_ENABLED = process.env.VT_ENABLED ? (process.env.VT_ENABLED === 'true') : true;
 const SSM_ENABLED =  process.env.SSM_ENABLED ? (process.env.SSM_ENABLED === 'true') : true;
 const API_ENABLED = process.env.API_ENABLED ? (process.env.API_ENABLED === 'true') : true;
+export const IN_PROD = process.env.IN_PROD ? process.env.IN_PROD === 'true' : false;;
 
 export const IN_CI = process.env.BZERO_IN_CI ? (process.env.IN_CI === '1') : false;
```
[the remark this answers is not in the document]
Comment (sidpremkumar) [reply src/system-tests/tests/system-test.ts:69]: https://github.com/bastionzero/zli/pull/405/commits/2add2eddc2ce727c4ff4be9e751a739a28397976

**Review by ymarcus93 — DISMISSED** [review]

**Review by tflpd — DISMISSED** [review]

**Review by tflpd — APPROVED** [review]

**Pull request merged**